ivy_tests/test_ivy/test_frontends/test_jax/test_random.py

```diff
@@ -1417,3 +1419,99 @@ def call():
     for u, v in zip(ret_np, ret_from_np):
         assert u.dtype == v.dtype
         assert u.shape == v.shape
+
+
+
+# Define the helper function to sample mean and covariance matrix
+def get_mean_cov_vector(shape, dtype):
+    mean_dim_size = shape[-1]
+
+    @st.composite
+    def mean_strategy(draw):
+        mean_size = draw(st.integers(min_value=1, max_value=mean_dim_size))
+        mean_values = draw(st.lists(st.floats(min_value=-10, max_value=10), min_size=mean_size, max_size=mean_size))
+        return np.array(mean_values, dtype=dtype)
+
+    mean = mean_strategy()
+
+    return mean
+
+# Sample the mean
+mean = get_mean_cov_vector(shape=(4,), dtype=np.float64)
```

Comment:
why is it required to be global?

Reply:
That was a mistake, I intended to call it within the scope of the test. therefore:
` mean=get_mean_cov_vector(shape=(4,), dtype=np.float64),
    cov=helpers.dtype_and_values(
        available_dtypes=("float32","float64"),
        min_value=0,
        max_value=10,
        shape=helpers.ints(min_value=2, max_value=5).map(lambda x: tuple([x, x])),`